index.html

```diff
@@ -28,10 +28,10 @@
             <div class="hero-normal">
                 Dedicated to making the world <strong>gooder</strong>
             </div>
-        </div>
-        <div class="hero-bottom center">
-          <a href="https://ti.to/codeforgood/rubybythebay" class="btn btn-huge btn-3d"><i class="fa fa-heart fa-1x"></i>Register</a>
-          <a href="https://ti.to/codeforgood/rubybythebay" class="btn btn-huge btn-3d"><i class="fa fa-heart fa-1x"></i>Sponsor an attendee</a>
+            <div class="hero-bottom center">
```

Comment:
@jtu0 I moved this part into the "here-content" div. It's not *quite* the right semantic naming anymore but I think it's reasonable enough.